sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/orc/OrcUtils.scala

```diff
@@ -543,3 +544,10 @@ object OrcUtils extends Logging {
     result
   }
 }
+
+object V2Column {
+  def unapply(e: org.apache.spark.sql.connector.expressions.Expression): Option[String] = e match {
```

Comment:
It seems cannot use this method.

Reply:
Can you show the errors you hit? It's not clear to me what's wrong with this object.